android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -0,0 +1,200 @@
+package org.fossasia.openevent.fragments;
+
+import android.animation.ObjectAnimator;
+import android.annotation.TargetApi;
+import android.app.Activity;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
```

Comment:
Please remove unused imports from everywhere asap. I'm gonna upload the apk to release.